### DISCUSSION TITLE
Liveview image is larger then the final Picture

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#228] [Q&A] [answered]
The image section in the live view is larger than in the still photo. I can't adjust anything on the camera and the webcam only has manual focus. Do you have an idea how I can adapt the live view to the still photo using software?

I know it is not a bug but i'll try to give the party people the chance to get in the right possition for the picture.

**Comment by mgineer85** [comment]
Yes, this would be an enhancement somebody needs to implement at one point. 
mit would need to crop the liveview to the actual input of the still.
That could be automated also.
Feel free to create a feature request and if you want to implement 😀

**Reply by Striker62** [reply]
;-) it is not a matter of willing .
This would be far away from my skills .

**Reply by mgineer85** [reply, marked as answer]
There is a kind of "workaround" available in next version:
In userdata/private.css add following:
```
/* zoom livestream */
#preview-stream {
    transform: scale(1.2);
}
```

This will zoom the livestream by factor 1.2 - adjust until it fits the still picture sufficiently well.

**Reply by Striker62** [reply]
thanks that is great . next version mean 2.02 ?

**Reply by mgineer85** [reply]
Yes

**Comment by Striker62** [comment]
that works great, your'e the best 👍
